docs/customization.md

```diff
@@ -122,3 +122,27 @@ For example, if you'd like to add your own styles for printing a page, you could
   }
 }
 ```
+
+## Override includes
+
+The site can be modified by overriding any of the custom includes provided by default in the theme.
+
+To do this, create an `_includes` directory and make a copy of the specific file you wish to modify. Any content added to this file will override the theme defaults. You can learn more about this process in the Jekyll docs for [Overriding theme defaults](https://jekyllrb.com/docs/themes/#overriding-theme-defaults).
+
+The following includes were made available to you:
```

Comment:
I wonder - should this be subheadings? Ex:

```
### Custom Footer

`/_includes/footer_custom.html`

This content appears at the bottom of every page's main content. More info for this include can be found in the [Configuration - Footer content](/docs/configuration/#footer-content).
```

What do you think?

Reply:
I've added headings as you described.